### PR TITLE
KNIGHT task 2 baseline

### PR DESCRIPTION
**Pull request by alex-golts**

### CODE REVIEW

File: fuse_examples/classification/knight/baseline/dataset.py [hunk 1]
```diff
@@ -71,7 +71,14 @@ def prepare_clinical(sample_dict: dict) -> dict:
 
 def knight_dataset(data_dir: str = 'data', cache_dir: str = 'cache', split: dict = None, \
     reset_cache: bool = False, post_cache_processing_func: Optional[Callable] = None, \
-        rand_gen = None, batch_size=8, resize_to=(256,256,110)):
+        rand_gen = None, batch_size=8, resize_to=(256,256,110), task_num=1):
+    if task_num == 1:
+        target_name='data.gt.gt_global.task_1_label'
```
Comment (mosheraboh) [inline fuse_examples/classification/knight/baseline/dataset.py:76]: Both task_num and target_name are defined in two different places - can we define all the per task variables in one place and pass it

**Review by mosheraboh — APPROVED** [review]
Great!
Just one comment inline

File: fuse_examples/classification/knight/baseline/dataset.py (hunk 1 above)
Comment (alex-golts) [reply fuse_examples/classification/knight/baseline/dataset.py:76]: I think I addressed it now

Comment (mosheraboh) [reply fuse_examples/classification/knight/baseline/dataset.py:76]: I meant to write num_classes and target_name.
After your last modifications. You still define num_classes and target_name twice, right?

Comment (alex-golts) [reply fuse_examples/classification/knight/baseline/dataset.py]: I see what you mean. please check now

**Review by mosheraboh — APPROVED** [review]
Great! Thanks!

**Pull request merged**